src/mapblock_mesh.cpp

```diff
@@ -242,7 +242,7 @@ static u16 getSmoothLightCombined(v3s16 p, MeshMakeData *data)
 
 	for (u32 i = 0; i < 8; i++)
 	{
-		const MapNode &n = data->m_vmanip.getNodeRefUnsafeCheckFlags(p - dirs8[i]);
+		MapNode n = data->m_vmanip.getNodeNoExNoEmerge(p - dirs8[i]);
```

Comment:
Because unlike `getNodeRefUnsafeCheckFlags`, `getNodeNoExNoEmerge` returns an object and not a reference. And, well, const-reference to a 4-byte object looks weird anyway.